virtcontainers/ich.go

```diff
@@ -0,0 +1,813 @@
+// Copyright (c) 2019 Ericsson Eurolab Deutschland GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"bufio"
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+	"fmt"
+	"net"
+	"syscall"
+
```

Comment:
Looking at this file with `git diff` shows a lot of extraneous whitespace (shows in red). I'd consider running `gofmt` over all the changed files.

Reply:
Done.

Reply:
As soon as this initial version of the driver isn't the subject of major requested structural changes, I'll start working on the unit tests for clh.go